nifi-commons/nifi-site-to-site-client/src/main/java/org/apache/nifi/remote/client/SiteToSiteClient.java

```diff
@@ -560,7 +560,8 @@ public SiteToSiteClient build() {
         }
 
         /**
-         * @return the configured URL for the remote NiFi instance
+         * @return the configured URL for the remote NiFi instance.
+         * The URL string possibly contains multiple URL entries in comma-separated format.
          */
         public String getUrl() {
```

Comment:
@ijokarumawak my concern is not code within the NiFi codebase, but rather code written by third parties. I believe there are others already who have code unrelated to NiFi, that actually depend on this client lib for sending/receiving data.